lib/style/module_directives.ex

```diff
@@ -229,9 +229,9 @@ defmodule Styler.Style.ModuleDirectives do
     end
   end
 
-  defp lift_aliases(aliases, requires, nondirectives) do
+  defp lift_aliases(aliases, requires, nondirectives, excluded_by_config) do
     excluded =
-      Enum.reduce(aliases, @stdlib, fn
+      Enum.reduce(aliases, MapSet.union(@stdlib, excluded_by_config), fn
```

Comment:
Do you think it would be worth pre-calculating this when Styler first initializes and putting it in `persistent_term` so that it only needs to be calculated once, since it won't vary over the course of a single formatter run?

Reply:
ooooooo i was sad about calculating it every time, but hadn't thought to globally cache it via persistent_term! that's a beaut. could do that with the whole config so that you don't have to pass it down thru context, oo la la